src/iuse_actor.cpp

```diff
@@ -1629,10 +1630,10 @@ long fireweapon_on_actor::use( player *p, item *it, bool t, const tripoint& ) co
         const auto tool = dynamic_cast<const it_tool *>( it->type );
         if( tool == nullptr ) {
             debugmsg( "Non-tool has fireweapon_on actor" );
-            it->make( "none" );
+            it->convert( "null" );
```

Comment:
Is "null" correct? I didn't test that one and it's a debug thing anyway, but just making sure it should be like that.